### DISCUSSION TITLE
漫蛙失效，大佬能修复一下吗

### DISCUSSION THREAD

**Discussion by As-sss** [xiaohucode/yidaRule#245] [General]
感谢大佬

**Comment by QAD21** [comment]
换一下域名就行了，不想换域名就用科学上网

**Reply by wuhao1115** [reply]
建立握手阶段发生问题是什么情况？

**Reply by QAD21** [reply]
？？？？

**Reply by wuhao1115** [reply]
我改了一下域名他显示这玩意

**Reply by 8721653** [reply]
锁定了没法改

**Comment by As-sss** [comment]
受累帮我改一下，我真的不会弄

发自我的iPhone


------------------ 原始邮件 ------------------
发件人: wuhao1115 ***@***.***&gt;
发送时间: 2024年11月18日 19:24
收件人: xiaohucode/yidaRule ***@***.***&gt;
抄送: As-sss ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [xiaohucode/yidaRule] 漫蛙失效，大佬能修复一下吗 (Discussion #245)
